cli/script/command-parser.ts

```diff
@@ -212,6 +212,12 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
     .command("logout", "Log out of the current session", (yargs: yargs.Argv) => {
         isValidCommandCategory = true;
         isValidCommand = true;
+        yargs.usage(USAGE_PREFIX + " logout [--local <true|false>]")
```

Comment:
Can we not specify <true|false> here? The convention for these flags is that if they exist, they change something, and if they don't, it's the default behavior